### PR TITLE
Display custom inbound authenticators in management console.

### PR DESCRIPTION
**Pull request by mevan-karu**
### Proposed changes in this pull request

Fix for issue : https://github.com/wso2/product-is/issues/8203

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java [hunk 1]
```diff
@@ -944,28 +945,32 @@ private boolean isCustomInboundAuthType(String authType) {
...
-
-        InboundAuthenticationRequestConfig[] authRequests = serviceProvider
-                .getInboundAuthenticationConfig()
-                .getInboundAuthenticationRequestConfigs();
-
-        if (authRequests != null) {
-            for (InboundAuthenticationRequestConfig request : authRequests) {
-                if (isCustomInboundAuthType(request.getInboundAuthType())) {
-                    inboundAuthenticationRequestConfigs.add(request);
+        if (customInboundAuthenticatorConfigs != null && customInboundAuthenticatorConfigs.size() > 0) {
```
Comment (emswbandara) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java:954]: we can use CollectionUtils.isNotEmpty(customInboundAuthenticatorConfigs) && CollectionUtils.isNotEmpty(inboundAuthenticationRequestConfigs) here and remove nested if

Comment (mevan-karu) [reply components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java:954]: If we remove the nested if condition and use a &&, we have to do the CollectionUtils.isNotEmpty(customInboundAuthenticatorConfigs)  check twice.

Comment (emswbandara) [reply components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java:954]: where do we check CollectionUtils.isNotEmpty(customInboundAuthenticatorConfigs)  for the second time? we can remove the second check

Comment (mevan-karu) [reply components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java:954]: If we remove the nested if, we have to check twice.

**Pull request merged**